pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -329,16 +329,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.PodInfo, p
 	defer p.lock.Unlock()
 	pod := pInfo.Pod
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod is already present in unschedulableQ")
+		return fmt.Errorf("pod: %v/%v is already present in unschedulable queue", pod.Namespace, pod.Name)
 	}
 
 	// Refresh the timestamp since the pod is re-added.
 	pInfo.Timestamp = p.clock.Now()
 	if _, exists, _ := p.activeQ.Get(pInfo); exists {
-		return fmt.Errorf("pod is already present in the activeQ")
+		return fmt.Errorf("pod is already present in the active queue")
```

Comment:
can you also print the pod namespace and name in the error message? Same for the backoff queue line below

Reply:
seems my latest commit did not send successfully

Updated it.